torch/csrc/jit/passes/op_replacement.cpp

```diff
@@ -0,0 +1,69 @@
+#include <c10/util/Exception.h>
```

Comment:
Should we consider renaming the file to reflect the pass name? op_replacement seems pretty/too generic

Reply:
sounds good